data/data-pipeline/data_pipeline/etl/score/etl_utils.py

```diff
@@ -305,3 +312,73 @@ def create_codebook(
     return merged_codebook_df[constants.CODEBOOK_COLUMNS].rename(
         columns={constants.CEJST_SCORE_COLUMN_NAME: "Description"}
     )
+
+
+def compare_to_list_of_expected_state_fips_codes(
+    actual_state_fips_codes: typing.List[str],
+    nation_expected: bool = True,
```

Comment:
I want this to be conus_expected but I understand why no normal person would do that.

Reply:
i kinda like conus_expected lol

Reply:
The problem - the nation is not actually the same as CONUS. Continental US does *not* include Alaska and Hawaii, right?

Reply:
i would actually do conus_expected AND not_conus_expected as flags maybe to make it explicit. A few datasets like NCLD are just conus, some datasets are conus, nconus, and PR, some are comprehensive, etc. In the same way you have pr / islands separated out, i think conus / nconus could be useful.

Reply:
done!